tests/node_compat/run_all_test_unmodified.ts

```diff
@@ -22,7 +22,9 @@ import {
   usesNodeTestModule,
 } from "./common.ts";
 import { generateTestSerialId } from "./test.ts";
+import { join } from "@std/path/join";
 
+const testSuitePath = "tests/node_compat/runner/suite";
```

Comment:
Maybe let's use `import.meta.resolve()` or `join(import.meta.url, ...)` to ensure it's actually independent where it's being run from?